### ISSUE TITLE
Ignore image owner like `daprio` and save the image with only the image name as prefix

### ISSUE THREAD

**Issue by mukundansundar** [dapr/installer-bundle#2]
## Expected Behavior 
file name of the image only contains the image name followed by the version and extension.

## Actual Behavior
file name of the image in the installer bundle contains the owner name(daprio) followed by image name (dapr) followed by version and extension. 

Following up on the discussion in this [issue](https://github.com/dapr/cli/issues/938), make the switch to save the image with filename that does not contain owner name as prefix.

**Comment by mukundansundar** [comment]
Let the details file look like this: 
```json
{
  "daprd": "1.7.0-rc.2", 
  "dashboard": "0.10.0-rc.1",
   "cli": "1.7.0-rc.2", 
  "daprBinarySubDir": "dist",
   "dockerImageSubDir": "docker",
  "daprImageName": "dapr/dapr:1.7.0-rc.2",
  "daprImageFileName": "dapr-dapr:1.7.0-rc.2.tar.gz"
}
```

**Comment by shivamkm07** [comment]
@artursouza this can be closed now since PR #3 fixing this issue is merged.
